### PR TITLE
Parse fixes

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: lib/openscap_parser/fix.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      @system ||= @parsed_xml['system']
+    end
+
+    def complexity
+      @complexity ||= @parsed_xml['complexity']
+    end
+
+    def text
+      @parsed_xml.text unless sub
+    end
```
Comment (akofink) [inline lib/openscap_parser/fix.rb]: Let's put this on the `XmlNode` as it applies to any xml node.

File: lib/openscap_parser/subs.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+module OpenscapParser
+  module Subs
+    def self.included(base)
+      base.class_eval do
+        def sub
+          return unless sub_node
+          Sub.new(parsed_xml: sub_node)
+        end
+
+        def sub_node
```
Comment (akofink) [inline lib/openscap_parser/subs.rb]: There can be 0 to n `sub` nodes according to the spec. IMO this should be pluralized

**Comment by xprazak2** [comment]
Multiple subs for one fixed are now parsed correctly.

File: lib/openscap_parser/fix.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    def system
+      @system ||= @parsed_xml['system']
+    end
+
+    def complexity
+      @complexity ||= @parsed_xml['complexity']
+    end
+
```
Comment (akofink) [inline lib/openscap_parser/fix.rb]: Should we include `disruption` and `strategy`?

File: lib/openscap_parser/sub.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+require 'openscap_parser/xml_node'
+
+module OpenscapParser
+  class Sub < XmlNode
+    def id
+      @id ||= @parsed_xml['idref']
+    end
+
```
Comment (akofink) [inline lib/openscap_parser/sub.rb]: Should we add `use` here?

> the <xccdf:sub> element MAY have a use attribute that indicates whether the <xccdf:Value> element's title or value should replace the <xccdf:sub> element.

**Comment by xprazak2** [comment]
I rebased and added parsing for `disruption`, `strategy` and `use`.

**Pull request merged**